routers/routers.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import APIRouter
+from fastapi import Depends, HTTPException
+from sqlalchemy.orm import Session
+from crud import crud
+from db import get_db
+from models.room import Room
+from schemas.schemas import RoomCreate, RoomTypeCreate
+
+router = APIRouter()
+
+
+@router.get("/rooms")
+async def get_all_rooms(
+    skip: int = 0, limit: int = 0, db: Session = Depends(get_db)
+):
+    rooms = crud.get_all_rooms(db=db, skip=skip, limit=limit)
```

Comment:
I think you need await here if you’re using async, but in general async is not really used in your code cause crud functions are synchronous